app/main.py

```diff
@@ -41,6 +40,5 @@ def wash_single_car(self, car: Car) -> None:
     def rate_service(self, rating: float) -> None:
         self.average_rating = round(
             ((self.average_rating * self.count_of_ratings) + rating)
-            / (self.count_of_ratings + 1), 1
-        )
+            / (self.count_of_ratings + 1), 1)
         self.count_of_ratings += 1
```

Comment:
Actually previous approach was better)
I meant to break down whole expression in a smaller parts, like:
```
change_count_of_wheels(self, added_wheels) -> None:
new_wheels_per_side = self.wheel_per_side + added_wheels
self.wheels_count = new_wheels_per_side * self.sides
```
instead of :
```
change_count_of_wheels(self, added_wheels) -> None:
new_wheels_per_side = (self.wheel_per_side + added_wheels) * self.sides
```
It's redundant in exact example, but I wanted to show the flow.